libs/framework/include/celix_launcher.h

```diff
@@ -68,7 +81,7 @@ int celixLauncher_launch(const char *configFile, celix_framework_t **framework);
  *
  * @param config FILE* to the config file (config.properties)
  * @param framework Output parameter for the framework.
- * @return CELIX_SUCCESS if successful.
+ * @return 0 if successful.
  */
 int celixLauncher_launchWithStream(FILE *config, celix_framework_t **framework);
```

Comment:
I found that its implementation is removed in 985c35f2f6ef6cd48ba63e8d117a928dfda3fe6a?
Should it be recovered or removed completely?

Reply:
I will reintroduce this function, so that we do not break previous releases.